lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -0,0 +1,302 @@
+                        this.message(message);
+                    } catch (error) {
+                        this.message(this.getMessage('error'));
+                    }
+                } else {
+                    this.message(this.getMessage('error'));
+                }
+            },
+        });
+
+        AccountSettingsFieldViews.ReadonlyFieldView = AccountSettingsFieldViews.FieldView.extend({
+
+            templateSelector: '#field_readonly-tpl',
+
+            initialize: function (options) {
+                this._super(options);
+                _.bindAll(this, 'render', 'updateValueInField');
+                this.listenTo(this.model, "change:" + this.options.valueAttribute, this.updateValueInField);
+            },
+
+            render: function () {
+                this.$el.html(this.template({
+                    title: this.options.title,
+                    value: this.modelValue(),
+                    message: this.helpMessage,
+                }));
+                return this;
+            },
+
+            updateValueInField: function () {
+                this.$('.account-settings-field-value').html(this.modelValue());
+            },
+        });
+
+        AccountSettingsFieldViews.TextFieldView = AccountSettingsFieldViews.FieldView.extend({
+
+            templateSelector: '#field_text-tpl',
+
+            events: {
+                'change input': 'saveValue',
+            },
+
+            initialize: function (options) {
+                this._super(options);
+                _.bindAll(this, 'render', 'fieldValue', 'updateValueInField', 'saveValue');
+                this.listenTo(this.model, "change:" + this.options.valueAttribute, this.updateValueInField);
+            },
+
+            render: function () {
+                this.$el.html(this.template({
+                    title: this.options.title,
+                    value: this.modelValue(),
+                    message: this.helpMessage,
+                }));
+                return this;
+            },
+
+            fieldValue: function () {
+                return this.$('.account-settings-field-value input').val();
+            },
+
+            updateValueInField: function () {
+                this.$('.account-settings-field-value input').val(this.modelValue() || '');
+            },
+
+            saveValue: function (event) {
+                var attributes = {};
+                attributes[this.options.valueAttribute] = this.fieldValue();
+                this.saveAttributes(attributes);
+
+            },
+        });
+
+        AccountSettingsFieldViews.EmailFieldView = AccountSettingsFieldViews.TextFieldView.extend({
+
+            successMessage: function() {
+                return this.indicators['success'] + interpolate_text(
+                    gettext('Complete you email change by clicking the confirmation link emailed to {new_email_address}.'),
```

Comment:
Missing "r" in "your".
"To complete your email address change, click the confirmation link sent via email to {new_email_address}."